.github/workflows/release.yml

```diff
@@ -32,20 +32,14 @@ jobs:
           force: ${{ github.event.inputs.force }}
           calver: true
       - name: Create new release in self-hosted Sentry
+        uses: getsentry/action-release@v1
         env:
-          SENTRY_DSN: 'https://19555c489ded4769978daae92f2346ca@self-hosted.getsentry.net/3'
```

Comment:
Is this set in `environment.production` as well?

Reply:
We actually don't need this to set up releases since that's used for monitoring events for a specific project.

Reply:
Just for reference, ran the entire workflow for this and it succeeded
https://self-hosted.getsentry.net/organizations/self-hosted/releases/22.10.1/?project=3